Police-Data-Accessibility-Project/data-sources-app-v2
# feature: main search route

Adds the initial search functionality to the main page of the data sources v2 app.

See https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/249

## Still to do
- [ ] Add unit tests for new components and utils 

## Open questions
#### Are we allowing users to search for multiple record categories or no?

## To test
1. Run the API locally
2. Run the client app locally
3. Perform a search
4. Check the console for the path to which we will eventually redirect for our search. Not that all search values are present in URL query params

It should look something like this:

https://github.com/user-attachments/assets/bec7314f-bf67-4246-b1e4-eaf6a277abef
